src/test/java/tests/login/LoginTest.java

```diff
@@ -0,0 +1,44 @@
+package tests.login;
+
+import framework.enums.Customers;
+import framework.models.Customer;
+import framework.pages.CustomerInfoPage;
+import framework.pages.HomePage;
+import framework.pages.LoginPage;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import tests.BaseTest;
+
+public class LoginTest extends BaseTest{
+
+    LoginPage loginPage;
+    HomePage homePage;
+    CustomerInfoPage customerInfoPage;
+    Customer customer;
+
+    @BeforeMethod
+    void setup() {
+        customer = Customers.TEST_USER.getCustomer();
+        loginPage = new LoginPage();
+        loginPage.openPage();
+        homePage = new HomePage();
+        customerInfoPage = new CustomerInfoPage();
+    }
+
+    @Test(description = "After login user returns to the Home Page", priority = 1)
+    void loginTest() {
+        loginPage.loginAs(customer);
+        SoftAssert softAssert = new SoftAssert();
+        softAssert.assertEquals(homePage.getCaption(), "Вітаємо в BUTLERS!", "Actual page caption is: " + homePage.getCaption());
```

Comment:
For assert message try to use String.format. It will look better.

Reply:
Created a variable "message".
Added String.format().